grpc-gateway/config.yaml

```diff
@@ -35,6 +35,18 @@ apis:
           certFile: "/secrets/public/cert.crt"
           useSystemCAPool: false
 clients:
+  authorizationServer:
```

Comment:
Please update also deployment config which is still part of this repo.

Reply:
@rafajpet FYI

Reply:
Could you please link it? I don't know what you mean.

Reply:
https://github.com/plgd-dev/cloud/blob/v2/docs/guide/deployment/grpc-gateway.md

Reply:
grpc-gw documentation updated